Source/WebCore/loader/FormSubmission.cpp

```diff
@@ -71,13 +71,13 @@ static void appendMailtoPostFormDataToURL(URL& url, const FormData& data, const
 
     if (equalLettersIgnoringASCIICase(encodingType, "text/plain")) {
         // Convention seems to be to decode, and s/&/\r\n/. Also, spaces are encoded as %20.
-        body = PAL::decodeURLEscapeSequences(body.replace('&', "\r\n"_s).replace('+', ' '));
+        body = PAL::decodeURLEscapeSequences(makeStringByReplacingAll(makeStringByReplacingAll(body, '&', "\r\n"_s), '+', ' '));
     }
 
     Vector<char> bodyData;
     bodyData.append("body=", 5);
     FormDataBuilder::encodeStringAsFormData(bodyData, body.utf8());
-    body = String(bodyData.data(), bodyData.size()).replace('+', "%20"_s);
+    body = makeStringByReplacingAll(String(bodyData.data(), bodyData.size()), '+', "%20"_s);
```

Comment:
Another case where we want a `StringView`.

Side thought: Would be nice if we didn’t have to copy this form body so many times, first to a `CString` to convert to UTF-8, then to a `String` to convert it to all-ASCII, the a new `String` if there’s an existing query to concatenate the new part, then build the `String` for storage in the `URL` inside `URL::setQuery`, and delete all the two intermediates. I guess it’s convenient to write the code this way, but I wish there was a simpler pipeline to get the characters into the URL. A form body might be something really large!